ClingoFluent command
---
bool success
string status
ClingoFluent[] observable_fluents
---
string status
ClingoFluent[] observable_fluents
